### ISSUE TITLE
Merriweather font not loading in Firefox

### ISSUE THREAD

**Issue by sheharyarn** [elixir-lang/ex_doc#695]
[Opening any HexDocs page](https://hexdocs.pm/elixir/Kernel.html) in Firefox doesn't load the Merriweather font, which causes it to look like this:

<img width="855" alt="screen shot 2017-03-29 at 6 59 36 pm" src="https://cloud.githubusercontent.com/assets/2355108/24480367/f048ee4c-14b1-11e7-98aa-1dd8412e06bb.png">

---

The developer console shows these errors:

<img width="1039" alt="screen shot 2017-03-29 at 7 02 12 pm" src="https://cloud.githubusercontent.com/assets/2355108/24480432/4f526206-14b2-11e7-926d-030ed815d15d.png">

```
downloadable font: Layout: DFLT script doesn't satisfy the spec. DefaultLangSys is NULL (font-family: "Merriweather" style:normal weight:300 stretch:normal src index:2) source: https://fonts.gstatic.com/s/merriweather/v15/ZvcMqxEwPfh2qDWBPxn6nkZRWJQ0UjzR2Uv6RollX_g.woff2  css:106:12
downloadable font: Layout: Failed to parse script table 0 (font-family: "Merriweather" style:normal weight:300 stretch:normal src index:2) source: https://fonts.gstatic.com/s/merriweather/v15/ZvcMqxEwPfh2qDWBPxn6nkZRWJQ0UjzR2Uv6RollX_g.woff2  css:106:12
downloadable font: GSUB: Failed to parse script list table (font-family: "Merriweather" style:normal weight:300 stretch:normal src index:2) source: https://fonts.gstatic.com/s/merriweather/v15/ZvcMqxEwPfh2qDWBPxn6nkZRWJQ0UjzR2Uv6RollX_g.woff2  css:106:12
downloadable font: rejected by sanitizer (font-family: "Merriweather" style:normal weight:300 stretch:normal src index:2) source: https://fonts.gstatic.com/s/merriweather/v15/ZvcMqxEwPfh2qDWBPxn6nkZRWJQ0UjzR2Uv6RollX_g.woff2  css:106:12
downloadable font: Layout: DFLT script doesn't satisfy the spec. DefaultLangSys is NULL (font-family: "Merriweather" style:italic weight:300 stretch:normal src index:2) source: https://fonts.gstatic.com/s/merriweather/v15/EYh7Vl4ywhowqULgRdYwIB0ue0Sk5cwvYx5tGiUAApw.woff2  css:138:12
downloadable font: Layout: Failed to parse script table 0 (font-family: "Merriweather" style:italic weight:300 stretch:normal src index:2) source: https://fonts.gstatic.com/s/merriweather/v15/EYh7Vl4ywhowqULgRdYwIB0ue0Sk5cwvYx5tGiUAApw.woff2  css:138:12
downloadable font: GSUB: Failed to parse script list table (font-family: "Merriweather" style:italic weight:300 stretch:normal src index:2) source: https://fonts.gstatic.com/s/merriweather/v15/EYh7Vl4ywhowqULgRdYwIB0ue0Sk5cwvYx5tGiUAApw.woff2  css:138:12
downloadable font: rejected by sanitizer (font-family: "Merriweather" style:italic weight:300 stretch:normal src index:2) source: https://fonts.gstatic.com/s/merriweather/v15/EYh7Vl4ywhowqULgRdYwIB0ue0Sk5cwvYx5tGiUAApw.woff2  css:138:12
```

---

I'm using:
 - Macbook Pro 2016
 - MacOS Sierra 10.12.13
 - Firefox Developer Edition - 54.0a2 (2017-03-27) (64-bit)

This only happens in Firefox (DE), and not Safari or Chrome.

**Comment by milmazz** [comment]
@sheharyarn It seems that Firefox (among others) is using [ots](https://github.com/khaledhosny/ots) to try to sanitize Open Type fonts. The *Merriweather* font is hosted in http://fonts.google.com. So, I think this issue should be addressed by the [Google Fonts team](https://github.com/google/fonts), wdyt?

/cc @josevalim @dignifiedquire

**Comment by michalmuskala** [comment]
The docs should work offline, though. So I'm not sure using google fonts is an option here.

**Comment by josevalim** [comment]
They do work offline because we fallback to a local font.

**Comment by aphillipo** [comment]
You can just download the font from Google and include it without too many issues; it's licensed as follows: 

http://scripts.sil.org/cms/scripts/page.php?site_id=nrsi&id=OFL_web

TLDR; "The OFL allows the licensed fonts to be used, studied, modified and
redistributed freely as long as they are not sold by themselves"

More about OTS and maybe changing to WOFF might help in Firefox?

http://typedrawers.com/discussion/1775/browser-ots-rejections-for-woff2-but-not-woff

**Comment by sheharyarn** [comment]
The problem seems to be with the `v2.001` update of the Merriweather font. There's an [issue open](https://github.com/EbenSorkin/Merriweather/issues/44) on the font's [Github repository](https://github.com/EbenSorkin/Merriweather) too.

**Comment by josevalim** [comment]
Ping. Is this fixed? Hopefully they rolled back the version? Thank you!

**Comment by aphillipo** [comment]
Firefox latest seems to work fine.  Testing the developer edition now...

**Comment by whatyouhide** [comment]
I am 👍 on a better stack :)
